templates/github/.github/workflows/scripts/before_install.sh.j2

```diff
@@ -87,22 +87,21 @@ cd ..
 {% endif %}
 
 {% if plugin_name != 'pulpcore' %}
+git clone --depth=1 https://github.com/pulp/pulp-smash.git
+
+if [ -n "$PULP_SMASH_PR_NUMBER" ]; then
+  cd pulp-smash
+  git fetch --depth=1 origin pull/$PULP_SMASH_PR_NUMBER/head:$PULP_SMASH_PR_NUMBER
+  git checkout $PULP_SMASH_PR_NUMBER
+  cd ..
+  pip install --upgrade --force-reinstall ./pulp-smash
+fi
+
 # When building a (release) tag, we don't need the development modules for the
 # build (they will be installed as dependencies of the plugin).
 if [ "${TAG_BUILD}" = "1" ]; then
```

Comment:
I think you can remove this if

Reply:
The entire thing?

Reply:
yes, from what I see, the if is just repeating:
```python
pip install --upgrade --force-reinstall ./pulp-smash
```